cli/ballerina-cliutils/src/main/ballerina/pull/ballerina.pull.bal

```diff
@@ -1,3 +1,5 @@
+package ballerina.pull;
```

Comment:
let's not use ballerina as a pkg name.

Reply:
Fixed with 9fcdece5175329f9bb65d427cb38db2c9f44d701